pcdsdevices/gasdet.py

```diff
@@ -0,0 +1,345 @@
+"""
+Ophyd devices for the acqiris digitizer systems
+"""
+import logging
+from ophyd import EpicsSignal, EpicsSignalRO, Device
+from ophyd import Component as Cpt
+from ophyd import FormattedComponent as FCpt
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+
+class acqiris_channel(Device):
```

Comment:
Python convention is:
```suggestion
class AcqirisChannel(Device):
```